app/pages/constants.ts

```diff
@@ -0,0 +1,113 @@
+export type Step =
+  | "housingStatus"
+  | "subcategoriesRadio"
+  | "eligibilities"
+  | "subcategories"
+  | "results";
+
+export interface ServiceCategory {
+  algoliaCategoryName: string;
+  sortBy24HourAvailability?: boolean;
+  disableGeoLocation?: boolean;
+  sortAlgoliaSubcategoryRefinements?: boolean;
+  id: string;
+  name: string;
+  abbreviatedName?: string;
+  slug: string;
+  steps: Step[];
+  subcategorySubheading: string;
+  icon: {
+    name: string;
+    provider: string;
+  };
+}
+
+const defaultSubheading =
+  "What are you currently looking for? Select all that apply.";
+
+export const CATEGORIES: Readonly<ServiceCategory[]> = [
+  {
+    algoliaCategoryName: "Arts, Culture & Identity",
+    id: "356",
+    name: "Arts, Culture & Identity",
+    slug: "arts-culture-identity",
+    steps: ["subcategories", "results"],
+    subcategorySubheading: defaultSubheading,
+    icon: {
+      name: "fa-palette",
+      provider: "fa",
+    },
+  },
+  {
+    algoliaCategoryName: "Children's Care",
```

Comment:
For some reason, the `/childrens-care/results` page does not have the map and the "no results" section